hwilib/devices/trezor.py

```diff
@@ -307,6 +330,32 @@ def backup_device(self, label='', passphrase=''):
     def close(self):
         self.client.close()
 
+    # Prompt for a pin on device
+    def prompt_pin(self):
+        self.client.init_device()
+        if not self.client.features.pin_protection:
+            raise DeviceAlreadyUnlockedError('This device does not need a PIN')
+        if self.client.features.pin_cached:
+            raise DeviceAlreadyUnlockedError('The PIN has already been sent to this device')
+        print('Use \'sendpin\' to provide the number positions for the PIN as displayed on your device\'s screen', file=sys.stderr)
+        print(PIN_MATRIX_DESCRIPTION, file=sys.stderr)
+        self.client.call_raw(proto.Ping(message=b'ping', button_protection=False, pin_protection=True, passphrase_protection=(self.password is not '')))
```

Comment:
Currently I would set it to False by default because it will trigger a `PassphraseRequest` which is not supported! 

As a note, passphrase_protection can default to true in the ping request, it will be ignored by the device.
It would have the benefit of relying on the state of the actual device rather than potentially faulty user input.

Reply:
I've changed it to `False`. To be more consistent across devices, I think it would be better to leave the passphrase stuff as is.